### DISCUSSION TITLE
SDK vs software versioning style

### DISCUSSION THREAD

**Discussion by mst3k4ever** [AgoraDoc/agora-style-guide#20] [Ideas]
I see "v3.1.0" when discussing Agora SDK versions. Most places (Microsoft, Apple, etc.) do not include the "v", because it is understood.

For browser versions, I am seeing just "Chrome 58" or "Edge 80", which looks better to me than "Chrome v58", but maybe that is personal preference.

Do they have to agree? Not necessarily.

We should figure this out and then put it in the style guide.

Cheers,

Bill

**Comment by DITAWriter** [comment]
I'll just add that this has been a long-standing Agora convention, though Bill is right in stating that many other major software firms do not use the leading "v" prior to a version number. In part, I think the idea has been to keep the leading "v" for consistency's sake with older Agora docs (especially in cases where features for multiple software versions are referenced on a single article), though there's definitely a case to be made for modernizing the docs and removing the leading "v" wherever it appears.
For Bill's reference, in terms of style, you should know about the firms that are peers to Agora, such Tencent, TokBox, and Twilio. Our aim is to be at least as good, if not better than them. :)
Cheers!     On Monday, September 28, 2020, 04:02:04 p.m. EDT, mst3k4ever <notifications@github.com> wrote:  
 
 


I see "v3.1.0" when discussing Agora SDK versions. Most places (Microsoft, Apple, etc.) do not include the "v", because it is understood.

For browser versions, I am seeing just "Chrome 58" or "Edge 80", which looks better to me than "Chrome v58", but maybe that is personal preference.

Do they have to agree? Not necessarily.

We should figure this out and then put it in the style guide.

Cheers,

Bill
